userpackage1/provider.go

```diff
@@ -0,0 +1,175 @@
+package userpackage1
+
+
+
+type User struct {
```

Comment:
this repeats the user structure from user package it's not needed here

Reply:
Removed